react/utils/routes.ts

```diff
@@ -107,17 +108,23 @@ export const fetchServerPage = async ({
   fetcher,
   path,
   query: rawQuery,
+  deviceInfo,
 }: {
   path: string
   query?: Record<string, string>
   fetcher: GlobalFetch['fetch']
+  deviceInfo?: DeviceInfo
 }): Promise<ParsedServerPageResponse> => {
   const parsedUrl = parse(path)
   parsedUrl.search = undefined
   parsedUrl.query = {
     ...rawQuery,
     __pickRuntime: runtimeFields,
+    ...(deviceInfo && {
+      __device: deviceInfo.type,
```

Comment:
is this a new querystring param? If yes, we need to add it to the CDN whitelist

Reply:
you mean this? https://github.com/vtex/render-server/pull/615/files#diff-2ce406e9cb901ca2f74fd823587990c7R5

Reply:
(narrator: he didn't mean that)

gonna talk to frederico about it